Extensions/Today/ButtonWithSublabel.swift

```diff
@@ -0,0 +1,60 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+
+class ButtonWithSublabel: UIButton {
+    lazy var subtitleLabel = UILabel()
+
+     lazy var label = UILabel()
+
+     required init?(coder aDecoder: NSCoder) {
+         fatalError("init(coder:) has not been implemented")
+     }
+
+     convenience init() {
+         self.init(frame: .zero)
+     }
+
+     override init(frame: CGRect) {
+         super.init(frame: frame)
+         performLayout()
+     }
+
+     fileprivate func performLayout() {
+         let titleLabel = self.label
```

Comment:
a) Wondering why do we need `titleLabel = self.label`? 
can we not just use `self.label`?